nifi-registry/nifi-registry-core/nifi-registry-flow-diff/src/main/java/org/apache/nifi/registry/flow/diff/StandardFlowComparator.java

```diff
@@ -624,4 +624,4 @@ private FlowDifference difference(final DifferenceType type, final VersionedComp
     private static interface ComponentComparator<T extends VersionedComponent> {
         void compare(T componentA, T componentB, Set<FlowDifference> differences);
     }
-}
+}
```

Comment:
Thanks for making the updates @emiliosetiadarma, can you revert this change as well?